backend/census_historical_migration/workbooklib/secondary_auditors.py

```diff
@@ -64,7 +64,15 @@
 
 
 def xform_address_state(secondary_auditors):
-    """Default missing address_state to GSA_MIGRATION"""
+    """
+    Default missing address_state to GSA_MIGRATION
+
+    We track all records related to a section that undergo transformation and
+    log these changes in change_records. However, we only save this data into
+    the InspectionRecord table if at least one of the records has been
+    modified by the transformation. If no records related to the given section
+    were modified, then we do not save change_records into InspectionRecord.
+    """
```

Comment:
The tracking should happen outside of the `if statement`

Reply:
All good. I see you addressed this in another PR. I belive the comment update was not meant for `xform_address_state` but for ` xform_missing_major_program` instead.